compiler/rustc_const_eval/src/interpret/eval_context.rs

```diff
@@ -46,6 +47,9 @@ pub struct InterpCx<'mir, 'tcx, M: Machine<'mir, 'tcx>> {
 
     /// The recursion limit (cached from `tcx.recursion_limit(())`)
     pub recursion_limit: Limit,
+
+    // reuse allocation for bit set
+    always_live_locals_cache: GrowableBitSet<mir::Local>,
```

Comment:
Don't know good place for this, any suggestions?

Reply:
This is probably the most global state you can add it to.

However note that this InterpCx is re-created per constant, so if there are many constants to evaluate, then each will still do their own thing. I am also concerned that we might keep large buffers around for longer than we have to.